pkg/microservice/aslan/core/environment/service/helm.go

```diff
@@ -47,14 +52,28 @@ import (
 	"github.com/koderover/zadig/pkg/types"
 	"github.com/koderover/zadig/pkg/util"
 	"github.com/koderover/zadig/pkg/util/converter"
+	jsonutil "github.com/koderover/zadig/pkg/util/json"
 )
 
+type HelmReleaseQueryArgs struct {
+	ProjectName string `json:"projectName"     form:"projectName"`
```

Comment:
Use snake_case in json tag?

Reply:
refactor json tag style in one-time modification